virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -80,6 +80,9 @@ The SVVP Certification applies to:
 
 //CNV-28732 Release note: NEW
 
+//CNV-29048 Release note: NEW Metrics name changes
+* Many OpenShift Virtualization metrics have changed. These changes could affect your custom dashboards. See link:https://access.redhat.com/articles/7028805[OpenShift Virtualization 4.14 metric changes] for details.
```

Comment:
Hi @aspauldi,  Please comment that the link includes both metrics that changed name and deprecated alerts that would change in 4.15.

Reply:
Hi @sradco, do  you mean that there are metrics whose names have changed in 4.14 and also alerts (not metrics) that are being deprecated in 4.14 and changed (but not removed) in 4.15? 

When I look [here](https://docs.google.com/spreadsheets/d/1buMYpxGK29VzgonbZaA4uimcJ02q5YrDVy80QQ5mulI/edit#gid=559835819), I see only a list of metric name changes in 4.14.

Reply:
Sorry, it's all about metrics. I made a mistake in the comment and updated it now

Reply:
How would metrics deprecated in 4.14 change in 4.15? Would the metrics still exist in 4.15 (removal would happen at some point in the future, after 4.15) but be renamed?